src/components/VideoCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { DotsThreeVertical } from "phosphor-react";
+import Popup from "./Popup";
+import ClickAwayListener from "react-click-away-listener";
```

Comment:
This is something new I got to know about how to hide the popup when the mouse leaves it !! Really good exploration Anamika!!